portal/portal-sprint-97.md

```diff
@@ -0,0 +1,43 @@
+## Usability improvements
+
+In this sprint, we have focused to improve overall usability and to make sure that users have the same capabilities to generate unique CashBoxIdentifications as they have when using ft-Helipad.
+
+## Features
+
+### Middleware Configuration
+
+#### Autogenerate unique CashBoxIdentification for QueuesDE (DE)
+
+In Germany it is required by law to use a vendor-unique identifier for the CashBoxIdentification. Now, the new functionality allows users to generate themselves a CashboxIdentification easily in the Portal. The generated identifier is based on the queueid and is globally unique.
+
+![CashBoxIdentification.png](images/sprint-97/CashBoxIdentification.png) 
+
+#### Reworked SignaturCreationUnit DE page (DE)
+
+The SignaturCreationUnit DE page was been reworked. Even though it appears to be similar with the old page, response times of the page are faster and the overall data overview is impoved. Aside from previous functionalities that are still there, some additional points can be found:
+
+Outlet number is shown inside of the table
+Serial number is moved to expander
+Certification identification and number of the signatures are added to expander
```

Comment:
Markdown blunder:
To have a line break in Markdown, a line has to end with a space ' '. Otherwise, it will continue without break.
Also, this looks suspiciously like a bullet list which can be achieved with a '- My Bullet Point'.

Reply:
fixed it, thanks!